internal/impl/sql/integration_test.go

```diff
@@ -396,13 +405,13 @@ processors:
 		var insertBatch service.MessageBatch
 		for i := 0; i < 10; i++ {
 			insertBatch = append(insertBatch, service.NewMessage([]byte(fmt.Sprintf(`{
-	"foo": "doc-%v",
-	"bar": %v,
+	"foo": "doc-%d",
+	"bar": %d,
 	"baz": "and this"
 }`, i, i))))
 		}
 		require.NoError(t, inFn(context.Background(), insertBatch))
-		require.NoError(t, streamIn.StopWithin(time.Second))
+		require.NoError(t, streamIn.StopWithin(15*time.Second))
```

Comment:
I hope this is reasonable. With one second, I get timeouts for MySQL and Oracle.